packages/ember-runtime/lib/mixins/target_action_support.js

```diff
@@ -60,10 +60,12 @@ export default Mixin.create({
   and target will be retrieved from properties of the object. For example:
 
   ```javascript
+  import { alias } from '@ember/object/computed';
+
   App.SaveButtonView = Ember.View.extend(Ember.TargetActionSupport, {
```

Comment:
I left `Ember.View.extend` here alone, since there isn't a documented import with RFC176. I assumed these types of examples could be filed as an issue and updated at a later date?

Reply:
yeah, Ember.View probably needs to go away, but that's for another pr...